1host.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	port: 3000,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 1 tab.](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
  port: 3000,
```